README.md

```diff
@@ -7,3 +7,74 @@ Project Proposal: https://docs.google.com/document/d/18x-RVWZ8EyQLAe7ZrwN4HEShcf
 - System Environment:           Dominic Pham
 - Functional Requirements:      Julia Chin / Dominic Pham
 - Non-functional Requirements:  Julia Chin / Dominic Pham
+
+# Delight
+# CS 157A – Team 18
+
+
+## Professor:
+Dr. Ching-Seh (Mike) Wu
+
+
+## Team Members:
+Julia Chin
+Kimleng Hor
+Dominic Pham
+
+### Project Overview
+
+Our team will be creating a website to help people find recipes for maintaining a healthy diet. Researchers have found that Americans eat out very often as opposed to cooking food at home. Hence, obesity is a problem in America. According to Vox, foods at restaurants have 20 to 40 percent more calories than home-cooked food. Our team wants to introduce a platform where people can choose their diets and be provided with recipes that suit their diets. With our platform, people will be able to learn how to eat right. Our stakeholders are people who are looking to learn how to maintain a healthy diet. This platform is important because it provides resources to guide people to living a healthy lifestyle.
+
+### Structure Diagram (3-tier Architecture)
+
+<img src="/images/StructureDiagram.png" width=200>
```

Comment:
Can we increase the size of the image? It looks pretty small. If you want to see how the file looks, look at this link: https://github.com/CS157A-Team18/CS157A-Team18/blob/1f48a91a6a2780aedae8cc28eed3c86545d42671/README.md

Reply:
Okay sure